macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -83,6 +83,11 @@
                 source_value,
                 bucket_start,
                 bucket_end,
+                {%- if metric_properties and metric_properties.seasonality and metric_properties.seasonality=='day_of_week' %}
+                    {{ elementary.edr_day_of_week_mod_7('bucket_end') }} as bucket_end_mod_7,
```

Comment:
Only open thing I'm wondering about - I have a column in `get_anomalies_scores_query` that has the meaning of "day_of_week(bucket_end) as an integer modulo 7" when we've got `seasonality: day_of_week` , but is just all `0` when we don't have seasonality. Wondering how to call it . Current name is `bucket_end_mod_7` .

Reply:
I think it's a pretty narrow name and implementation. 
If we want the flexibility to support other seasonality configs in the future, I suggest:

1. Naming the column `bucket_seasonality` 
I think it would also be much clearer to save there `Sunday` instead of 1, and null instead of 0. 

2. Set a var in the top like:
{%- set seasonality_expression = get_bucket_seasonality() %}
And do the logic of returning 0/null or an expression there (easier to expand later and also looks clearer, I think).

Reply:
@RoiTabach You didn't address this